actix-http/src/lib.rs

```diff
@@ -31,7 +31,7 @@
 #![doc(html_favicon_url = "https://actix.rs/favicon.ico")]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
 
-pub use http::{uri, uri::Uri, Method, StatusCode, Version};
+pub use http::{uri, uri::Uri, Method, StatusCode, Version, status::InvalidStatusCode};
```

Comment:
I'm fine re-exporting this, but I think it belongs in the `error` module.

Reply:
Great. Ok moved.